docs/install.md

```diff
@@ -7,12 +7,16 @@
 
 ## Requirements
 
-- [Kubernetes](https://kubernetes.io/) v1.9 or later
+- [Kubernetes](https://kubernetes.io/) v1.9 or later with an SSD storage class
+  - Cluster role administrator access
```

Comment:
I would link to https://kubernetes.io/docs/reference/access-authn-authz/rbac/ here (I can't directly link to `cluster-admin`, but there are a number of references to it on this page)